crates/data-structures/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license-file = "LICENSE"
 description = "Assorted data structures used in spacetimedb"
 
 [features]
-serde = ["dep:serde"]
+serde = ["dep:serde", "hashbrown/serde"]
```

Comment:
why is this feature needed now?

Reply:
Same reason - no longer being enabled by some other crate now that `core` isn't in the build graph.